setup.cfg

```diff
@@ -65,19 +112,14 @@ vagrant =
 windows =
     pywinrm
 
-[pbr]
-skip_authors = True
-skip_changelog = True
-warnerrors = True
-
-[entry_points]
+[options.entry_points]
 console_scripts =
-    molecule = molecule.shell:main
+    molecule = molecule.__main__:main
+
+[options.packages.find]
+where = .
 
 [build_sphinx]
 all_files = 1
 build-dir = doc/build
 source-dir = doc/source
```

Comment:
We can also add `warning-is-error = True` here too?

Reply:
Yes, but it's not related to this packaging change. So you can send another PR if you want.